src/Goteo/Entity/DataSet.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/*
+ * This file is part of the Goteo Package.
+ *
+ * (c) Platoniq y Fundación Goteo <fundacion@goteo.org>
+ *
+ * For the full copyright and license information, please view the README.md
+ * and LICENSE files that was distributed with this source code.
+ */
+
+namespace Goteo\Entity;
+
+use DateTime;
+use Goteo\Application\Config;
+use Goteo\Model\Image;
+
+class DataSet {
+
+    const DATASET_TYPE_PROJECTS = 'projects';
+    const DATASET_TYPE_INVESTS = 'invests';
```

Comment:
These 2 constants could be renamed to TYPE_PROJECTS and TYPE_INVESTS. Since they are in the DataSet entity class, we already have the context over there and repeating it in the name of the constant is redundant.